src/main/scripts/startMZmine_Windows.bat

```diff
@@ -52,16 +52,6 @@ mkdir %TMP_FILE_DIRECTORY%
 
 rem Set R environment variables.
 set R_HOME=C:\Program Files\R\R-2.12.0

```

Comment:
Yes, I did. But in some cases it can be required (if auto-detection fails). That is why I kept it for now. At the time I started working on all this you had an excellent suggestion which was more or less: "Remove it completely from the scripts, and, in case auto-detection failed, ask the user to give a path via the Project > Set Preferences panel".

I had no time to implement it, but the idea still stands (got it written on one of my several post-it :).